packages/playwright/src/index.ts

```diff
@@ -369,7 +369,9 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
     });
 
     let counter = 0;
-    const closeReason = testInfo.status === 'timedOut' ? 'Test timeout of ' + testInfo.timeout + 'ms exceeded.' : 'Test ended.';
+    let closeReason = testInfo.status === 'timedOut' ? 'Test timeout of ' + testInfo.timeout + 'ms exceeded.' : 'Test ended.';
+    if (testInfo.status === 'failed' && testInfo.error?.message?.startsWith('TimeoutError: Step timeout of '))
```

Comment:
We don't do this for `page.click` that timed out, so I don't think we should do it for step either.